model-optimizer/requirements_dev.txt

```diff
@@ -1,9 +1,5 @@
 coverage==4.4.2
-m2r==0.1.12
 pyenchant==1.6.11
-isort==4.3.21
-astroid==2.1.0
-pylint==2.1.1
-Sphinx==1.6.5
+pylint==2.5.0
```

Comment:
@ababushk, will CI use pylint version from requirements or we need a reflective change in CI?

Reply:
PyLint on GitHub actions use requirements.txt directly from repo - and it has failed due to this I bet.
Jenkins doesn't use pylint from requirements.txt when it runs `static-analysis` job. Actually, pylint execution can be removed from Jenkins as it is redundant due to GitHub actions implementation.

Reply:
PyLint on GitHub actions use requirements.txt directly from repo - and it has failed due to this I bet.
Jenkins doesn't use pylint from requirements.txt when it runs `static-analysis` job. Actually, pylint execution can be removed from Jenkins as it is redundant due to GitHub actions implementation.

Reply:
I resolve the issue since it does not affect this PR.